Sources/StreamChatUI/UIConfig.swift

```diff
@@ -111,15 +111,15 @@ public extension UIConfig {
 
 public extension UIConfig {
     struct Font {
-        public var captionBold: UIFont = .streamCaptionBold
-        public var footnote: UIFont = .streamFootnote
-        public var footnoteBold: UIFont = .streamFootnoteBold
-        public var body: UIFont = .streamBody
-        public var bodyBold: UIFont = .streamBodyBold
-        public var bodyItalic: UIFont = .streamBodyItalic
-        public var headline: UIFont = .streamHeadline
-        public var headlineBold: UIFont = .streamHeadlineBold
-        public var title: UIFont = .streamTitle
+        public var footnoteBold = UIFont.preferredFont(forTextStyle: .footnote).bold
+        public var subheadline = UIFont.preferredFont(forTextStyle: .subheadline)
+        public var subheadlineBold = UIFont.preferredFont(forTextStyle: .subheadline).bold
+        public var body = UIFont.preferredFont(forTextStyle: .body).withWeight(.medium)
+        public var bodyBold = UIFont.preferredFont(forTextStyle: .body).bold
+        public var bodyItalic = UIFont.preferredFont(forTextStyle: .body).italic
+        public var headline = UIFont.preferredFont(forTextStyle: .headline)
+        public var headlineBold = UIFont.preferredFont(forTextStyle: .headline).bold
+        public var title = UIFont.preferredFont(forTextStyle: .title1)
```

Comment:
Okay I am kinda confused and I don' understand this.

- I understand our demo designs are bit different from Apple guidelines. 
- However I don't understand this change uses prefferedFonts and not the `stream` ones. Could you please document this/describe this to me? 😅